crates/oxc_minifier/src/peephole/remove_dead_code.rs

```diff
@@ -537,6 +564,18 @@ impl<'a, 'b> PeepholeOptimizations {
             _ => false,
         }
     }
+
+    fn is_empty_constructor(constructor: &MethodDefinition<'a>) -> bool {
```

Comment:
nit: maybe call this `is_empty_method_definition` or perhaps assert that `constructor.kind` is `MethodDefinitionKind` or have the check in here?